AuxTel/AuxTel-Troubleshooting/ATCS-Troubleshooting/ATDome-shutter-fails-to-close.rst

```diff
@@ -0,0 +1,108 @@
+.. This is a template for troubleshooting when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
```

Comment:
This is a duplicated effort from SITCOM-1361. Maybe @isotuela  can fix it later.

Reply:
@isotuela- can you?